### ISSUE TITLE
[use-cases] We need to consistently use our terms

### ISSUE THREAD

**Issue by halindrome** [opencreds/website#58]
There are a lot of terms.  For the sake of our readers, we need to use them consistently.  And we should use as few unique ones as possible!  I propose that we use these:

issuer - the entity that issues the credential.
holder - the entity that the credential is about, and who "holds" onto it.
consumer - the entity that needs to receive and analyze a credential.

Let's avoid terms like "recipient" - that has a flow feel to it and will change depending on how the data is flowing.

Thoughts @gkellogg, @burnburn, @bsletten ?

**Comment by burnburn** [comment]
Yes, I didn't like recipient myself, but it was the term used most consistently in the use cases I saw, so I continued using it.  Your new suggestions are fine.  Once you say go I'm happy to convert all the terms in my section to match what you list above.

**Comment by halindrome** [comment]
Fair enough.  While you are waiting, if you look in ../common/terms.html there are a bunch of terms defined.  I imagine we could define more, but maybe it would be worth a review for everyone.

**Comment by bsletten** [comment]
Thanks for all the consolidation work, Shane. I am speaking all day today but will do a pull tonight and hopefully a bit of work tomorrow morning.
